polar-core/tests/integration_tests.rs

```diff
@@ -49,8 +49,8 @@ fn no_isa(_: Term, _: Symbol) -> bool {
     true
 }
 
-fn no_is_subspecializer(_: u64, _: Symbol, _: Symbol) -> bool {
-    false
+fn no_is_subspecializer(_: u64, a: Symbol, b: Symbol) -> bool {
+    a == b
```

Comment:
Can take this out but it seems to make sense

Reply:
In the host libraries, `is_subspecializer` returns false for equivalent classes. A class A is only a subspecializer if it is a subclass of B w.r.t. the MRO of an instance i. So I think this should return false even for equivalent classes.

You can see the implementation in python in host.py, I just checked this for reference.